expressEngine/customer/service/customer.js

```diff
@@ -6,31 +6,51 @@ import { Customer } from '../../model';
 
 // import messages
 import { error, success } from '../../cms/customer';
+// import helper
+import Helper from '../../lib/helper';
```

Comment:
@rahuljainss create a baseHelper and use that

Reply:
Sir done with the changes please review the changes